receiver/postgresqlreceiver/metadata.yaml

```
@@ -0,0 +1,69 @@
+name: postgresql
+
+labels:
+  database:
+    description: The name of the database.
+  table:
+    description: The schema name followed by the table name.
+  source:
+    description: The block read source type.
+    enum: [ heap_read, heap_hit, idx_read, idx_hit, toast_read, toast_hit, tidx_read, tidx_hit ]  
+  operation:
+    description: The database operation.
+    enum: [ seq, seq_tup_read, idx, idx_tup_fetch ]
+  state:
+    description: The tuple (row) state either dead or live
```

```suggestion
    description: The tuple (row) state.
```

Since the values are enumerated in a separate field, we shouldn't need to include them in the description.

---

fixed